testing/tests/DevExpress.ui.widgets.editors/htmlEditorParts/mentionIntegration.tests.js

```diff
@@ -58,13 +55,7 @@ module('Mentions integration', {
                 .dxHtmlEditor('instance');
         };
 
-        this.addText = (element, text, prevText) => {
-            if(IS_IE11) {
-                element.outerText = `${prevText}${text}`;
-            } else {
-                element.innerText += text;
-            }
-        };
+        this.addText = (element, text, prevText) => element.innerText += text;
```

Comment:
I think we should remove this function and refactor places where it was used to make the code more clear

Reply:
done